lib/ansible/utils/module_docs_fragments/azure.py

```diff
@@ -72,8 +72,9 @@ class ModuleDocFragment(object):
     - "azure >= 2.0.0"
 
 notes:
-    - For authentication with Azure you can pass parameters, set environment variables or use a profile stored
-      in ~/.azure/credentials. Authentication is possible using a service principal or Active Directory user.
+    - For authentication with Azure you can pass parameters, set environment variables, use a profile stored
+      in ~/.azure/credentials or login with Azure CLI.
```

Comment:
`C(~/.azure/credentials)` will give formatted text

http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#formatting-options

Could you please update the other docs, for example `C(AZURE_SUBSCRIPTION_ID)`, etc?

Thanks in advance